src/metrics/chainflip/eventsRotationInfo.ts

```diff
@@ -0,0 +1,73 @@
+import promClient, {Counter, Gauge} from "prom-client";
+import {Context} from "../../lib/interfaces";
+import {FlipConfig} from "../../config/interfaces";
+import {decodeAddress} from "@polkadot/util-crypto";
+import makeRpcRequest from "../../utils/makeRpcRequest"
+
+
+const metricName: string = "cf_rotation_phase_attempts";
+const metric: Counter = new promClient.Counter({
```

Comment:
Lets name these according to the other metrics. i.e. metricNameRotationPhaseAttempts